test/lifecycle_test/component2.dart

```diff
@@ -20,16 +20,17 @@ class _SetStateTest extends react.Component2 with LifecycleTestHelper {
       };
 
   @override
-  void init() => lifecycleCall('init');
-
-  @override
-  getInitialState() => {
-        'counter': 1,
-        'shouldThrow': true,
-        'errorFromGetDerivedState': '',
-        'error': '',
-        'info': '',
-      };
+  void init() {
+    lifecycleCall('init');
+    this.initializeState({
```

Comment:
#nit Using `this.` is discouraged when it isn't needed: https://dart.dev/guides/language/effective-dart/usage#dont-use-this-when-not-needed-to-avoid-shadowing

No worries, though; we should get more lints turned on in this repo so these kinds of checks can be automated

Reply:
💯